qbraid/runtime/batch.py

```
@@ -0,0 +1,355 @@
+            _active_batch.reset(self._token)
+            self._token = None
+        if self._session_token is not None:
+            _active_batch_session.reset(self._session_token)
+            self._session_token = None
+
+        # Close the batch on the backend
+        if self._batch_data is not None:
+            try:
+                self._batch_data = self.client.close_batch(self._batch_data.batchJobQrn)
+                logger.info("Closed batch session: %s", self._batch_data.batchJobQrn)
+            except Exception as err:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)